include/quicly/cc.h

```diff
@@ -35,6 +35,13 @@ extern "C" {
 #include <string.h>
 #include "quicly/constants.h"
 
+typedef enum {
+    /**
+     * Reno, with 0.7 beta reduction
+     */
+    CC_RENO_MODIFIED
+} quicly_congestion_controller_t;
```

Comment:
`quicly_cc_type_t`? Having `quicly_cc_t` and `quicly_congestion_controller_t` is confusing.